packages/peregrine/lib/talons/Region/useRegion.js

```diff
@@ -1,21 +1,37 @@
+import { useEffect, useRef } from 'react';
 import { useQuery } from '@apollo/react-hooks';
-import { useFieldState } from 'informed';
+import { useFieldApi, useFieldState } from 'informed';
 
 export const useRegion = props => {
     const {
         countryCodeField = 'country',
+        field = 'region',
         optionValueKey = 'code',
         queries: { getRegionsQuery }
     } = props;
 
+    const hasInitialized = useRef(false);
     const countryFieldState = useFieldState(countryCodeField);
     const { value: country } = countryFieldState;
+    const regionFieldApi = useFieldApi(field);
+
+    // Reset region value when country changes. Because of how Informed sets initialValues,
+    // we want to skip the first state change of the value being initialized.
+    useEffect(() => {
+        if (country) {
+            if (hasInitialized.current) {
+                regionFieldApi.setValue();
```

Comment:
Maybe indicate this is a reset with a comment.

Reply:
`reset()` from the `fieldApi` had the behavior I wanted, so I made that change in ab0adc9, which should be more self documenting. Thanks for pointing out there was much more to that API than the docs lead you to believe.